src/objects/DataImportBatch__c.object

```diff
@@ -257,6 +257,17 @@
         <trackTrending>false</trackTrending>
         <type>Currency</type>
     </fields>
+    <fields>
+        <fullName>GiftBatch__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>This flag is set during batch creation to use the gift entry batch feature.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>This batch is intended for use with Batch Gift Entry. Batches are automatically marked as gift batches when they are set up through the Gift Batches tab. Manually changing this field could have unintended consequences.</inlineHelpText>
```

Comment:
@smarovich Can you review this help text? This field replaces "Use Batch Gift Entry Grid" but I changed the help text.

Reply:
Is the tab still "Batch Gift Entry"?

Reply:
Yes! Good catch.

Reply:
Sorry for the delay. The help text looks awesome! Nicely done. I would change the description slightly to "This flag is set during batch creation and indicates that the batch is intended for use with Batch Gift Entry."